### ISSUE TITLE
Processing seems broken (again?)

### ISSUE THREAD

**Issue by maxdevjs** [NixOS/nixpkgs#73900]
**Describe the bug**

This issue has been previously referenced in #66518 and should have been fixed in #66740 . I tried to install Processing today and I get the same error described in #66518 .

```
hash mismatch in fixed-output derivation '/nix/store/74jkcqrddp7nxijr3ha09d2mnfck5836-reference.zip':
  wanted: sha256:0ai0cr62gc7n6y22ki3qibyj1qnlaxv1miqxmmahfk3hpbyfqz9n
  got:    sha256:198bpk8mzns6w5h0zdf50wr6iv7sgdi6v7jznj5rbsnpgyilxz35

cannot build derivation '/nix/store/bp16h84nmjx2fysz2ikaz41i5kw9hbwm-processing3-3.4.drv': 1 dependencies couldn't be built
```

**Metadata**
Please run `nix run nixpkgs.nix-info -c nix-info -m` and paste the result.

```
- system: `"x86_64-linux"`
 - host os: `Linux 4.19.77, NixOS, 19.03.173575.0e0ee084d6d (Koi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2.2`
 - channels(max): `""`
 - channels(root): `"nixos-19.03.173663.6d448d7fbd6"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Basically I get the old `3-3.4` and not the new `3.5.3`.

How can I have the fixed version? Is it not available for channel 19.03 (I already updated the channel)?

**Comment by kolbycrouch** [comment]
Perhaps the hash wasn't updated to the reflect the version. Try copy-pasting the "got" hash into the expression for "processing" in your NIX_PATH and see if that works for you.

**Comment by maxdevjs** [comment]
Thank you @kolbycrouch , I had to switch to channel 19.09 to get the fixed version :)

**Comment by stephane-rolland** [comment]
@maxdevjs is it still ok for you ? Tried on the 19.09 in the current state and it still fails the same way. Gonna update the channel to check if that's better.

**Comment by maxdevjs** [comment]
@stephane-rolland sorry for the late reply. 

No, for what I know it broke again a couple days after it was fixed (#66740, [my comment](https://github.com/NixOS/nixpkgs/pull/66740#issuecomment-557867646)) and still it is in `19.09` (tried right now).

I am not using `NixOS`, but what I did back then has been to use `steam-run` (installed `steam-run-native`).

**Comment by stephane-rolland** [comment]
@maxdevjs  sorry I did not find a solution. But I did not tried very hard. I wanted just to give Processing a try, and discovered that Processing Python module is based on obsolete Python 2.7 / Jython , so it was not I wanted.
